src/SharedFrameworkValidation/SharedFrameworkValidation.proj

```diff
@@ -13,25 +13,52 @@
   </Target>
 
-  <Target Name="BuildPackagesProject">
-    <PropertyGroup>
-      <_BuildPackagesProjectCommand>"$(DotnetToolCommand)" build $(MSBuildThisFileDirectory)/RestoreSDKProject/RestoreSDKProject.csproj /p:_PublishRID=$(_PublishRID) /p:_MetaPackageDestinationFolder=$(SharedFrameworkValidationRefPath)</_BuildPackagesProjectCommand>
-    </PropertyGroup>
-    <Exec Command="$(_BuildPackagesProjectCommand)" />
+  <Target Name="BuildDependenciesProject">
+    <MSBuild Projects="../../external/SharedFrameworkValidation/harvestPackages.SharedFramework.depproj"
+             Properties="_PublishRID=$(_PublishRID)"
+             ContinueOnError="ErrorAndStop" />
   </Target>
 
-  <Target Name="PublishPackagesProject">
+  <Target Name="RunApiCompat" DependsOnTargets="BuildDependenciesProject">
     <PropertyGroup>
-      <_PublishPackagesProjectCommand>"$(DotnetToolCommand)" publish $(MSBuildThisFileDirectory)/RestoreSDKProject/RestoreSDKProject.csproj /p:_PublishRID=$(_PublishRID) /p:_MetaPackageDestinationFolder=$(SharedFrameworkValidationRefPath) -f netcoreapp2.0 -r $(_PublishRID) -o $(SharedFrameworkValidationRuntimePath)</_PublishPackagesProjectCommand>
+      <ApiCompatCmd>$(ToolHostCmd) "$(ToolsDir)ApiCompat.exe"</ApiCompatCmd>
     </PropertyGroup>
-    <Exec Command="$(_PublishPackagesProjectCommand)" />
+
+    <MakeDir Directories="$(IntermediateOutputPath)" />
+
+    <!-- Microsoft.NETCore.App Ref vs. runtime packages -->
+    <Exec Command="$(ApiCompatCmd) &quot;$(SharedFrameworkValidationRefPath)&quot; $(ApiCompatArgs) -implDirs:&quot;$(SharedFrameworkValidationRuntimePath)&quot;"
+          CustomErrorRegularExpression="Total Issues: [^0]"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="Low" />
+    
+    <!-- Netstandard.Library vs. Microsoft.NETCore.App Ref -->
+    <Exec Command="$(ApiCompatCmd) &quot;$(NetStandardRefPath)/netstandard.dll&quot; $(ApiCompatArgs) -implDirs:&quot;$(SharedFrameworkValidationRefPath)&quot;"
+          CustomErrorRegularExpression="Total Issues: [^0]"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="Low" />
+
+    <!-- Last shipped Microsoft.NETCore.App Ref vs. current Microsoft.NETCore.App Ref -->
+    <Exec Command="$(ApiCompatCmd) &quot;$(RefRootPath)SharedFrameworkValidation.LKG&quot; $(ApiCompatArgs) -implDirs:&quot;$(SharedFrameworkValidationRefPath)&quot;"
+          CustomErrorRegularExpression="Total Issues: [^0]"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="Low" />
+
+    <!-- Last shipped runtime packages vs current runtime packages -->
+    <Exec Command="$(ApiCompatCmd) &quot;$(BinDir)runtime/SharedFrameworkValidation.LKG&quot; $(ApiCompatArgs) -implDirs:&quot;$(SharedFrameworkValidationRuntimePath)&quot; -baseline:&quot;$(MSBuildThisFileDirectory)ApiCompatBaseline.LKGRuntime.CurrentRuntime.txt&quot;"
```

Comment:
If we do that, then we wouldn't catch if there is a new break(although I know it's unlikely). Given that this will run in some official legs only, that means that we would never really know the output of this baseline as nobody will be updating it, so basically running ApiCompat on it would be useless, and in the unlikely case where there is a removed Api that breaks it, we would never catch it.